### PR TITLE
[WebInterface] add email confirm message

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
supabase migration changes
requires https://github.com/Drakkar-Software/OctoBot/pull/2343

### CODE REVIEW

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

File: Trading/Mode/grid_trading_mode/tests/test_grid_trading_mode.py [hunk 1]
```diff
@@ -371,8 +359,265 @@ async def test_create_orders_with_fixed_volume_per_order():
...
+        )
+        _check_created_orders(producer, trading_api.get_open_orders(exchange_manager), 100)
+
+
+async def test_start_after_offline_full_sell_side_filled_orders_price_back():
+    symbol = "BTC/USDT"
+    async with _get_tools(symbol) as (producer, _, exchange_manager):
+        # first start: setup orders
+        producer.sell_funds = decimal.Decimal("1")  # 20 orders
+        producer.buy_funds = decimal.Decimal("1")  # 24 orders
```
Comment (Herklos) [inline Trading/Mode/grid_trading_mode/tests/test_grid_trading_mode.py:543]: What does `1` means here?

**Review by Herklos — COMMENTED** [review]
Nice restoration mechanism!

File: Trading/Mode/grid_trading_mode/tests/test_grid_trading_mode.py (hunk 1 above)
Comment (GuillaumeDSM) [reply Trading/Mode/grid_trading_mode/tests/test_grid_trading_mode.py:543]: 1 BTC, it's kinda random, the idea is to allow as many orders as possible

Comment (Herklos) [reply Trading/Mode/grid_trading_mode/tests/test_grid_trading_mode.py]: Oh okay, I didn't expect this to be a quantity of BTC 😄

**Review by Herklos — APPROVED** [review]

**Pull request merged**